azurecontainerapps.ts

```diff
@@ -606,12 +606,12 @@ export class azurecontainerapps {
         }
 
         const environmentVariables: string = this.toolHelper.getInput('environmentVariables', false);
-
+        const isCappUpdateCommandUsed: boolean = this.noIngressUpdate || (!this.noIngressUpdate && !this.adminCredentialsProvided)
```

Comment:
Could you please share more info about that part of the change? Why do we need this change?

Reply:
Yes, this check was added because we had two different ways to add env vars when running containerapp create, up, and update. For `update`, the option name was `---replace-env-vars`. And for any other commands we used, it was `--env-vars`.

Due to one of the recent changes we made with ingress update (Issue: https://github.com/microsoft/azure-container-apps/issues/821 PR: #81), the wrong argument was appended to `update` command, which resulted in some user running into issue when they have ingress and env variables set.

And as this check was named, it checks if the situation is going to use `update` command. If it is, then `--replace-env-vars` option will be appended.

Reply:
OK, thanks for sharing more info!